test/unit/basic.js

```diff
@@ -0,0 +1,84 @@
+module( "basic", {
+	teardown: moduleTeardown
+} );
+
+if ( jQuery.css ) {
+	test( "css", function() {
+		expect( 18 );
```

Comment:
This already looks like too much... I'd expect to see one `test` per module (as you have), but no more than 10 assertions per test, and hopefully even fewer. For the CSS module, maybe one assertion like `assert.equal( div.css( "width", "50px" ).css( "width" ), "50px", ".css getter/setter" )` and one for `.show`/`.hide` and call it done.